libs/ui-web/package.json

```diff
@@ -21,7 +21,9 @@
   "peerDependencies": {
     "@angular/common": "^9.0.0",
```

Comment:
Why here are `@angular` modules version 9 ?

Reply:
Package updated to v10.